components/airtable/sources/common.js

```diff
@@ -0,0 +1,23 @@
+const airtable = require("../airtable.app");
+
+module.exports = {
+  props: {
+    airtable,
+    db: "$.service.db",
+    timer: {
+      type: "$.interface.timer",
+      default: {
+        intervalSeconds: 60 * 5,
+      },
+    },
+  },
+  hooks: {
+    activate() {
```

Comment:
btw, we've recently shipped a `deploy` hook that runs only at deploy time (instead of on every component update). [See this source as an example](https://github.com/PipedreamHQ/pipedream/blob/446b92a5173b550cc89145b0e6285c6f1b8a637b/components/uservoice/sources/new-nps-ratings/new-nps-ratings.js#L24).

Reply:
Yeah I noticed that one, but I don't think it applies here.
If a user decided to disable this event source, and then later re-enables it, we want to make sure that we update the start timestamp so that we don't capture any data generated during the period in which this event source was disabled.

Reply:
yeah this comment was more of an aside. I just wasn't sure if you knew about the `deploy` hook.